shell/platform/android/platform_view_android.cc

```diff
@@ -41,6 +41,7 @@ std::unique_ptr<AndroidSurface> AndroidSurfaceFactoryImpl::CreateSurface() {
     case AndroidRenderingAPI::kSoftware:
       return std::make_unique<AndroidSurfaceSoftware>();
     case AndroidRenderingAPI::kOpenGLES:
+      FML_DLOG(ERROR) << "using opengl";
```

Comment:
Remove this and change the "Using the Impeller rendering backend" log in `android_surface_gl_impeller.cc` to mention OpenGL ES